tests/test_guarani_minimum_score.py

```diff
@@ -1,33 +1,51 @@
 import json
 from pathlib import Path
+from collections import defaultdict
 
 
-DATA_DIR = Path("data/processed")
-MIN_GUARANI_SCORE = 0.70
-GUARANI_LANG_CODES = {"grn"}
+def test_all_documents_have_minimum_guarani_score():
+    DATA_DIR = Path(__file__).parent.parent / "data" / "processed"
+    MIN_GUARANI_SCORE = 0.7
+    GUARANI_LANG_CODES = {"grn", "gug", "gn"}
 
+    stats = defaultdict(lambda: {"total": 0, "failed": 0})
 
-def test_all_documents_have_minimum_guarani_score():
-    errors = []
+    jsonl_files = list(DATA_DIR.rglob("*.jsonl"))
+    assert jsonl_files, "No JSONL files found under data/processed/"
+
+    for jsonl_path in jsonl_files:
+        corpus_name = jsonl_path.parent.name
 
-    for jsonl_path in DATA_DIR.rglob("*.jsonl"):
         with jsonl_path.open(encoding="utf-8") as f:
-            for line_num, line in enumerate(f, start=1):
+            for line in f:
                 obj = json.loads(line)
 
-                if obj["language"] not in GUARANI_LANG_CODES:
-                    errors.append(
-                        f"{jsonl_path} line {line_num}: invalid language "
-                        f"{obj['language']}"
-                    )
-                    continue
-
-                if obj["language_score"] < MIN_GUARANI_SCORE:
-                    errors.append(
-                        f"{jsonl_path} line {line_num}: "
-                        f"language_score={obj['language_score']}"
-                    )
-
-    assert not errors, (
-        "Documents below minimum Guarani threshold:\n" + "\n".join(errors)
+                if obj["language"] in GUARANI_LANG_CODES:
```

Comment:
The code below has several blank lines that can be removed, improving readability and making the code more compact.